plugins/chat/assets/javascripts/discourse/routes/chat.js

```diff
@@ -68,6 +80,20 @@ export default class ChatRoute extends DiscourseRoute {
   }
 
   deactivate(transition) {
+    withPluginApi("1.8.0", (api) => {
+      api.setSidebarPanel("main");
+
+      if (getUserChatSeparateSidebarMode(this.currentUser).fullscreen) {
+        api.setCombinedSidebarMode();
+        api.showSidebarSwitchPanelButtons();
+      } else if (getUserChatSeparateSidebarMode(this.currentUser).always) {
+        api.setSeparatedSidebarMode();
+      } else {
+        api.setCombinedSidebarMode();
+        api.hideSidebarSwitchPanelButtons();
+      }
+    });
```

Comment:
We can extract a variable:

```suggestion
      const chatSeparateSidebarMode = getUserChatSeparateSidebarMode(this.currentUser);
      if (chatSeparateSidebarMode.fullscreen) {
        api.setCombinedSidebarMode();
        api.showSidebarSwitchPanelButtons();
      } else if (chatSeparateSidebarMode.always) {
        api.setSeparatedSidebarMode();
      } else {
        api.setCombinedSidebarMode();
        api.hideSidebarSwitchPanelButtons();
      }
    });
```

Reply:
Or I would even introduce a property on this route, given we access it in one more place (in the `activate()` method above).